santiao-guillen/homework-11/to-do-list-router/src/Components/Homework2.js

```diff
@@ -0,0 +1,16 @@
+
+const Homework2 = () => {
```

Comment:
@santiagoGuillenGlob , this approach is really wrong, your basically repeating the code in different components, you should use only one Homework component and pass props with the info to it